src/components/adjudications/AdjudicationTableRow.jsx

```diff
@@ -6,61 +6,50 @@ import LensIcon from '@material-ui/icons/Lens';
 
 import AdjudicationDialog from './AdjudicationDialog';
 
-class AdjudicationTableRow extends React.Component {
-  state = {};
-
-  render() {
-    const {
-      id,
-      judgeName,
-      audio,
-      specialAward,
-      choreoAward,
-      cumulativeMark
-    } = this.props;
-
-    return (
-      <TableRow key={id}>
-        <TableCell>
-          {judgeName}
-        </TableCell>
-        <TableCell>
-          <p>
-            {audio ? 'yes' : 'no'}
-          </p>
-        </TableCell>
-        <TableCell>
-          {cumulativeMark}
-        </TableCell>
-        <TableCell>
-          {specialAward && <LensIcon fontSize="inherit" style={{ color: 'purple' }} />}
-          {choreoAward && <LensIcon fontSize="inherit" color="primary" />}
-        </TableCell>
-        <TableCell>
-          <AdjudicationDialog currentValues={this.props} />
-        </TableCell>
-      </TableRow>
-    );
-  }
-}
-
-AdjudicationTableRow.propTypes = {
-  audio: PropTypes.string,
-  choreoAward: PropTypes.bool,
-  cumulativeMark: PropTypes.number,
-  id: PropTypes.number,
-  judgeName: PropTypes.string,
-  specialAward: PropTypes.bool
+const AdjudicationTableRow = ({
+  collectionName,
+  currentValues,
+  id
+}) => {
+  const {
+    audio,
+    choreoAward,
+    cumulativeMark,
+    judgeName,
+    specialAward
+  } = currentValues;
+
+  return (
+    <TableRow key={id}>
+      <TableCell>
+        {judgeName}
+      </TableCell>
+      <TableCell>
+        <p>
+          {audio ? 'yes' : 'no'}
+        </p>
+      </TableCell>
+      <TableCell>
+        {cumulativeMark}
+      </TableCell>
+      <TableCell>
+        {specialAward && <LensIcon fontSize="inherit" color="primary" />}
+        {choreoAward && <LensIcon fontSize="inherit" style={{ color: 'purple' }} />}
```

Comment:
We just swapped the colors here, was that intended? `specialAward` used to have the color purple styling, and now it has the `primary` color tag.

Reply:
@YangStephen yep that was intended, I realized I swapped the colors up, from the zeplin designs `specialAward` is red (`primary` color tag) and `choreoAward` is purple